TestVectorsAwsCryptographicMaterialProviders/dafny/TestVectorsAwsCryptographicMaterialProviders/src/VectorsComposition/AllRawAES.dfy

```diff
@@ -149,12 +149,28 @@ module {:options "-functionSyntax:4"} AllRawAES {
           decryptDescription := keyDescription
         )
 
+  const TestsWithMultipleUTF8Ec :=
+    set
+      keyDescription <- KeyDescriptions + KeyDescriptionsWithPsi,
+      algorithmSuite <- AllAlgorithmSuites.AllAlgorithmSuites,
+      commitmentPolicy | commitmentPolicy == AllAlgorithmSuites.GetCompatibleCommitmentPolicy(algorithmSuite),
+      encryptionContext <- EncryptionContextUtils.multipleEntriesUTF8EncryptionContext
+      ::
+        TestVectors.PositiveEncryptKeyringVector(
+          name := "Generated RawAES Multiple UTF8 Entries EC " + keyDescription.AES.keyId,
+          encryptionContext := encryptionContext,
+          commitmentPolicy := commitmentPolicy,
+          algorithmSuite := algorithmSuite,
+          encryptDescription := keyDescription,
+          decryptDescription := keyDescription
+        )
+
   const Tests :=
     {}
     + TestsNoEc
     + TestsWitPsiEc
     + TestsBasicEc
     + TestControlEc
-    + TestsWithDiffUTF8Ec
```

Comment:
Should we clean up the definition for this? Is it still used elsewhere?

Reply:
(Non-blocking)

Reply:
It is not used anywhere, we can completely remove it or leave it in for when Dafny doesnt overflow with all the tests.